packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -182,65 +229,84 @@ export const makeScheduler = async (
 
       void E(timer).cancel(stepCancelToken);
     }
+
+    if (schedule) {
+      publishSchedule(time);
+    }
   };
 
-  const scheduleRound = time => {
-    trace('nextRound', time);
+  // schedule the wakeups for the steps of this round
+  const scheduleSteps = now => {
```

Comment:
thanks for making clear that the time passed in is "now" versus some future scheduled time.

please do the same for `clockTick = (time, schedule)` so that it can end with,
```js
    if (schedule) {
      publishSchedule(now);
    }
```

Reply:
separately, passing around `now` feels like it leaves room for bugs. "now" is a shared truth.

WDYT of keeping `let now` in closure and just updating it whenever it changes? then each function can get its `now` from closure instead of params. If you do that, I think maybe the publishSchedule calls can get refactored to happen in fewer places.

Reply:
GH had this comment hidden so I didn't it before writing https://github.com/Agoric/agoric-sdk/pull/7339#discussion_r1160279248

I'll resolve this thread.